solr/solrj/src/java/org/apache/solr/common/params/CommonParams.java

```diff
@@ -160,6 +160,26 @@
    */
   String TIME_ALLOWED = "timeAllowed";
 
+  /**
+   * Is the query cancellable?
+   */
+  String IS_QUERY_CANCELLABLE = "canCancel";
+
+  /**
+   * Custom query UUID if provided.
+   */
+  String CUSTOM_QUERY_UUID = "queryUUID";
+
+  /**
+   * UUID for query to be cancelled
+   */
+  String QUERY_CANCELLATION_UUID = "cancelUUID";
```

Comment:
Hmm. Why do we need to use a separate parameter name? We either submit a new query (and it's handled by one handler) or cancel a query (and it's handled by a completely separate handler) so there's no danger of confusion. I think we should just use `queryUUID` in all places.